data/autoyast_xen/sles12sp4PV_PRG.xml

```diff
@@ -1094,36 +1051,13 @@ DefaultPolicy default
     </yesno_messages>
   </report>
   <services-manager>
-    <default_target>graphical</default_target>
+    <default_target>multi-user</default_target>
     <services>
-      <disable config:type="list"/>
+      <disable config:type="list">
+      <service>SuSEfirewall2</service>
+      </disable>
       <enable config:type="list">
-        <service>apparmor</service>
-        <service>btrfsmaintenance-refresh</service>
-        <service>cron</service>
-        <service>display-manager</service>
-        <service>haveged</service>
-        <service>irqbalance</service>
-        <service>iscsi</service>
-        <service>mcelog</service>
-        <service>nscd</service>
-        <service>ntpd</service>
-        <service>postfix</service>
-        <service>purge-kernels</service>
-        <service>rollback</service>
-        <service>rsyslog</service>
-        <service>smartd</service>
         <service>sshd</service>
-        <service>SuSEfirewall2</service>
-        <service>SuSEfirewall2_init</service>
-        <service>wicked</service>
```

Comment:
Isn't `wicked` required for the networking?

Reply:
It's required and enabled. We don't need to explicitly enable it in autoyast profile